packages/customer-account-ui-extensions/src/extension-points/types.ts

```diff
@@ -55,3 +89,16 @@ type ExtractedApiFromRenderExtension<T> = T extends RenderExtension<
 export type ApiForRenderExtension<
   ID extends keyof RenderExtensions
 > = ExtractedApiFromRenderExtension<RenderExtensions[ID]>;
+
+/**
+ * For a given run extension point, returns the type of the API that the
+ * extension will receive at runtime. This API type is the first argument to
+ * the callback for that extension point.
+ */
+export type ApiForRunExtension<
```

Comment:
Check if we can remove this and also `ApiForRenderExtension` now that we do have `ApiForExtension`